lib/interceptor.js

```diff
@@ -32,8 +32,9 @@ var interceptor = {
           payload = arguments[0];
         } else if (arguments[0] instanceof FormData) {
           payload = {};
-          for (var entry of arguments[0].entries()) {
-            payload[entry[0]] = entry.slice(1);
+          var entries = arguments[0].entries();
+          while(item = entries.next(), !item.done) {
```

Comment:
This is failing the code quality check. Adding a `var` before `item =` should do the trick :)